internal/controller/vrg_kubeobjects.go

```diff
@@ -328,7 +312,12 @@ func (v *VRGInstance) kubeObjectsGroupCapture(
 			return
 		}
 
-		captureInProgressStatusUpdate()
+		if v.instance.Spec.ReplicationState == ramen.Secondary &&
+			v.instance.Spec.Action == ramen.VRGActionRelocate {
```

Comment:
To keep the behviour of the previous code, remove the check for action == relocate.

To keep the code simpler and easier to follow, it will help to more the condition into v.kubeObjectsCaptureStatusFalse - we can always call it and it will do the right thing based on the vrg replication state.